README.md

```diff
@@ -6,25 +6,36 @@ It is fully free and fully open source. The license is Apache 2.0, meaning you a
+This plugin uses Pulsar Client 2.10.2. For broker compatibility, see the official Pulsar compatibility reference. If the compatibility wiki is not up-to-date, please contact Pulsar support/community to confirm compatibility.
 
 If you require features not yet available in this plugin (including client version upgrades), please file an issue with details about what you need.
 
 # Pulsar Input Configuration Options
 This plugin supports these configuration options. 
 
-| Settings    | Input type     | Required  |
-| ------------- |:-------------:| -----:|
-| codec      | string, one of ["plain","json"] | No |
-| topics      | array | Yes |
-| subscriptionName      | string | Yes |
-| consumerName      | string | No |
-| subscriptionType      | string, one of["Shared","Exclusive","Failover","Key_shared"] | No |
-| subscriptionInitialPosition| string, one of["Latest","Earliest"] | No |
+| Settings                         |                          Input type                          |  Required |
+|----------------------------------|:------------------------------------------------------------:|----------:|
+| serviceUrl                       |                            string                            |       Yes |
+| topics                           |                            array                             |       Yes |
+| subscriptionName                 |                            string                            |       Yes |
+| consumerName                     |                            string                            |       Yes |
+| subscriptionType                 | string, one of["Shared","Exclusive","Failover","Key_shared"] |        No |
+| subscriptionInitialPosition      |             string, one of["Latest","Earliest"]              |        No |
+| codec                            |               string, one of ["plain","json"]                |        No |
+| protocols                        |           array, ciphers list. default is TLSv1.2            |        No |
+| ciphers                          |                      array,ciphers list                      |        No |
+| enable_tls                       |       boolean, one of [true, false]. default is false        |        No |
+| tls_trust_store_path             |        string, required if enable_tls is set to true         |        No |
+| tls_trust_store_password         |                   string, default is empty                   |        No |
+| enable_tls_hostname_verification |       boolean, one of [true, false]. default is false        |        No |
+| allow_tls_insecure_connection    |        boolean, one of [true, false].default is false        |        No |
+| enable_token                     |        boolean, one of [true, false].default is false        |        No |
```

Comment:
Rename to `enable_auth`?

Reply:
Maybe rename it to `enable_jwt` more make sense

Reply:
This is used for multiple authentication plugins, so as the token, oauth2, tls, and basic, so on.